storage/postgres/postgres.go

```diff
@@ -325,7 +326,18 @@ func (p *Postgres) ReadAuthorizationModel(ctx context.Context, store string, mod
 		return nil, storage.ErrNotFound
 	}
 
+	// Update the schema version lazily if it is not a valid typesystem.SchemaVersion.
+	if version != typesystem.SchemaVersion1_0 && version != typesystem.SchemaVersion1_1 {
+		version = typesystem.SchemaVersion1_0
+		_, err = p.pool.Exec(ctx, "UPDATE authorization_model SET schema_version = $1 WHERE store = $2 AND authorization_model_id = $3", version, store, modelID)
+		if err != nil {
+			// Don't worry if we error, we'll update it lazily next time, but let's log:
+			p.logger.Warn("failed to lazily update schema version", zap.String("store", store), zap.String("authorization_model_id", modelID))
```

Comment:
Let's set this at the Error level, instead of warning. This shouldn't fail, and if it does we should know about it.

Reply:
That's pretty serious. Won't that wake up the on-call primary? Is there a way to log an error and not wake people up?